src/main/java/com/lambdaschool/oktafoundation/services/ApplicationServiceImpl.java

```diff
@@ -155,17 +162,14 @@ public Application updateAppStatus(Application application, long id)
         return apprepos.save(currentApp);
     }
 
-
-
-
     @Transactional
     @Override
     public Application saveByAuth(User user, Application application)
     {
         Application newApp = new Application();
-        User dbuser = userrepos.findById(user.getUserid())
-            .orElseThrow(() -> new ResourceNotFoundException("User id " + user.getUserid() + " not found"));
-        newApp.setUser(dbuser);
+//        User dbuser = userrepos.findById(user.getUserid())
+//            .orElseThrow(() -> new ResourceNotFoundException("User id " + user.getUserid() + " not found"));
+//        newApp.setUser(dbuser);
```

Comment:
I don't think we can comment out these lines. Commenting out 166 and 167 removes the 'ByAuth' part of `saveByAuth` and 168 removes the association the application has with the user that made it.

It should work fine if we just un-comment the code and let it run.